src/presentation/facades/controller-facade.ts

```diff
@@ -22,23 +23,23 @@ export class ControllerFacade {
       ServiceFacade.createRundownService(),
       RepositoryFacade.createRundownRepository(),
       ServiceFacade.createIngestService(),
-      new ExpressErrorHandler()
+      new ExpressErrorHandler(new LoggerService())
```

Comment:
There are being created new logger instances in a lot of places. I'd like this to be in one place, such that it can be used as a singleton. Currently, we are logging to the console, but if we were to change the LoggerService to write to a file on disk, all the instances would fight for write access to the file.

Reply:
Has been changed to a singleton

Reply:
We might be able to say YAGNI and not have it singleton at the moment, but as soon as we begin to write logs to a file, we would end up with multiple instances fighting for access to the samme file.

This is atleast how i understood it, when i asked Anders.